Makefile.js

```diff
@@ -181,10 +181,14 @@ function generateFormatterExamples(formatterInfo, prereleaseVersion) {
  * @returns {void}
  */
 function generateRuleIndexPage() {
-    const outputFile = "../website/_data/rules.yml",
+    const legacyWebsiteOutputFile = "../website/_data/rules.yml",
+        docsSiteOutputFile = "docs/src/_data/rules.json",
+        docsSiteMetaOutputFile = "docs/src/_data/rules_meta.json",
```

Comment:
I believe the changes in these two files will not be committed during the release. Our release process commits only [changelog](https://github.com/eslint/eslint-release/blob/198c7e700e501f2762f41396635b74fdb7d04e06/lib/release-ops.js#L381-L382) and [version](https://github.com/eslint/eslint-release/blob/198c7e700e501f2762f41396635b74fdb7d04e06/lib/release-ops.js#L385) (`npm version` creates a version commit and tag).

Reply:
Good call. Maybe we should update eslint-release to run an `npm build` command and then commit the result?

Reply:
> Good call. Maybe we should update eslint-release to run an `npm build` command and then commit the result?

Maybe we could add this as an option in [`generateRelease()`](https://github.com/eslint/eslint-release/blob/198c7e700e501f2762f41396635b74fdb7d04e06/lib/release-ops.js#L365)? Some projects don't have a `build` script (for example, [`eslint-plugin-markdown`](https://github.com/eslint/eslint-plugin-markdown/blob/87c2b536fd80b15e134766d92b90048ae45cbe1f/package.json#L22-L32)), and some project have a `build` script that should not be run for this purpose (for example, [`espree`](https://github.com/eslint/espree/blob/c98a7e0be7e21f28a00868660d66bda9d323f1d3/package.json#L70)).

Reply:
Alternatively, we could update and commit these files before running [`ReleaseOps.generateRelease()`](https://github.com/eslint/eslint/blob/ce035e5fac632ba8d4f1860f92465f22d6b44d42/Makefile.js#L267). A downside is that the commit would be included in the changelog.

Or add a `preversion` or `version` script ([npm version](https://docs.npmjs.com/cli/v8/commands/npm-version)). A downside is that this would bundle the changes into the version commit, I think.

Reply:
I’ll dig into this more.

Reply:
> we can add the new files and amend the commit after we generate the release.

Version tag will still point to the original version commit, which won't be in the history of the `main` branch anymore. I think that's not a common practice.

Reply:
Hmm good point. I suppose the next logical step is just to update the tag, too.